### ISSUE TITLE
IRI 1.5.0 is out.  Update to latest

### ISSUE THREAD

**Issue by tuddman** [muXxer/rock64-docker-iota-fullnode#1]
[IRI 1.5.0](https://github.com/iotaledger/iri/releases/tag/v1.5.0)

First, thank you for this!!    This repo has been very helpful in setting up a cluster of rock64's to run IOTA and some other apps I'm working on.

I've 'built' the latest docker container for IRI v.1.5.0  in the dockerfile.

Might you consider updating & pushing the [hub IRI](https://hub.docker.com/r/muxxer/rock64_iota_iri/)  as well to the latest?   Make it easy for people to get up and running.

Thank again for this great effort!

**Comment by muXxer** [comment]
Hi tuddman,

thank you very much! I wasn't aware that my repo is actually used, so I was busy with testing and developing CarrIOTA Hercules alpha in the last days.

I saw your issue and your forked repo after I actually updated my repo by myself. Haha sorry for that! :)
I will update the containers in 10 minutes, after my Rock64 is done with compiling RocksDB. :)
Thank you again for the feedback and the Issue report!
